src/Models/LoginSession.php

```diff
@@ -96,6 +97,14 @@ class LoginSession extends DataObject
      */
     private static $default_session_lifetime = 3600;
```

Comment:
We need to update the documentation for this to indicate that _technically_ the lifetime is actually this value plus the new last_accessed_threshold value.